### DISCUSSION TITLE
How is the Video stream Played?

### DISCUSSION THREAD

**Discussion by Mopele** [pystardust/ani-cli#1233] [General] [answered]
I am thinking about implementing a similar project for a diffrent website (no anime) and are wondering how it is possible to play the HLS(.m3u8) Streams. I know you are doing it with IINA, but when I extract the URL from allanime (the .m3u8 url)  and paste it into the IINA player I get an error. So how is it done in your case

**Comment by CoolnsX** [comment, marked as answer]
We pass the same user agent that was used to fetch the links in video player
